wiki/en/en-Client-Troubleshooting.md

```diff
@@ -18,7 +18,7 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 You can test whether you are hearing your signal correctly by doing the following:
 
-1. Get your audio into your computer.
+1. Connect your audio source (mic, instrument...) to your computer.
```

Comment:
remark-lint reported some issues here. I think we might need to go through the whole wiki and fix syntax errors. 
That's nothing you should need to worry about now.

Reply:
Yeah, I didn't have a clue what it was on about.